packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+* Fix mechanism to detect identifier in multiline WebVTT captions.
```

Comment:
This will need a version update (2.9.3)

Reply:
here's a link to the changelog style guide https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changelog-style

probably just need to change it to `Fixes`